lib/mixlib/install/version.rb

```diff
@@ -1,5 +1,5 @@
 module Mixlib
   class Install
-    VERSION = "0.8.0"
+    VERSION = "1.0.0"
```

Comment:
I am not sure if we should bump to 1.0.0 at this point. Since we are pretty much breaking the backwards compatibility I think we should do a major version bump. Any thoughts?